src/index.ts

```diff
@@ -26,16 +27,13 @@ const rules = readdirSync(join(__dirname, 'rules'))
   )
   .reduce(
     (acc, curr) =>
-      Object.assign(acc, { [curr]: importDefault(`./rules/${curr}`) }),
+      Object.assign(acc, { [curr]: importDefault(join(rulesDir, curr)) }),
     {},
   );
-let allRules = {};
-Object.keys(rules).forEach(function(key) {
-  allRules[`jest/${key}`] = 'error';
-});
+const allRules: Record<string, string> = {};
+Object.keys(rules).forEach(key => (allRules[`jest/${key}`] = 'error'));
 
-// eslint-disable-next-line import/no-commonjs
-module.exports = {
+export default {
```

Comment:
as mentioned in the other review, I don't think this can be `export default` as eslint just does `require('plugin')` and expect that object to have `rules` etc. I don't think it deals with `.default` etc

Reply:
Maybe, but the `lib` output looks the same:

`exports.default`:
```
Object.keys(rules).forEach(key => allRules[`jest/${key}`] = 'error');
exports.default = {
  configs: {
    all: {
  ....
```

`export default`:
```
Object.keys(rules).forEach(key => allRules[`jest/${key}`] = 'error');
var _default = {
  configs: {
   .... 
}
exports.default = _default;
```

Reply:
it was `module.exports`, not `exports.default `previously

Reply:
No idea what you mean - totally didn't blunder it and use `exports.default` by mistake ;P 

~But yeah still the same output as before:~

Yeah you're right. I'll revert that :joy:

Reply:
😀